src/sdk/messages/MessageType.ts

```diff
@@ -49,4 +49,9 @@ export enum MessageType {
    * Message sent from the addon
    */
   OAUTH_DIALOG_COMPLETED = 'cxt:auth:complete',
+
+  /**
+   * Message sent from the addon containing the data to enhance rich text editor content
+   */
+  ENHANCE_TEXT_EDITOR = 'ctx:enhance:text-editor',
```

Comment:
It is just a key but others are cxt not ctx

Reply:
I should probably get checked for dyslexia 😅